api/backup.go

```diff
@@ -187,9 +188,10 @@ type BackupSettingUpsert struct {
 	EnvironmentId int
 
 	// Domain specific fields
-	Enabled   bool `jsonapi:"attr,enabled"`
-	Hour      int  `jsonapi:"attr,hour"`
-	DayOfWeek int  `jsonapi:"attr,dayOfWeek"`
+	Enabled   bool   `jsonapi:"attr,enabled"`
+	Hour      int    `jsonapi:"attr,hour"`
+	DayOfWeek int    `jsonapi:"attr,dayOfWeek"`
+	HookURL   string `jsonapi:"attr,hook_url"`
```

Comment:
Please add a comment here as well. Also since this is in the code, we can add more implementation details, like which http method we are going to use.